ui/pages/bridge/bridge.util.ts

```diff
@@ -159,3 +210,29 @@ export async function fetchBridgeTokens(
   });
   return transformedTokens;
 }
+
+/**
+ * A function to return tx for setting allowance to 0 for USDT on Ethereum
+ *
+ * @param approval - The original transaction params for the required allowance
+ * @returns Modified approval transaction params that will reset allowance to 0
+ */
+export const getEthUsdtApproveResetTxParams = (approval: TransactionParams) => {
+  const UsdtContractInterface = new Contract(
+    ETH_USDT_ADDRESS,
+    ETHEREUM_USDT_APPROVALS_ABI,
+  ).interface;
```

Comment:
Can we use `abiERC20` from @metamask/metamask-eth-abis instead so we don't need to copy in the ABI?

Reply:
Done